### DISCUSSION TITLE
positive current, negative voltage

### DISCUSSION THREAD

**Discussion by akariwrynn** [pybamm-team/PyBaMM#2294] [General]
Hi,
I asked a question regarding applying a sinusoidal current as  (params['Current function [A]'] = lambda t: amplitude * pybamm.sin(2 * np.pi* freq[i]  * t). However, it seems that when the current is positive, the voltage becomes negative. I can basicly multiply with minus but do you know why this is happening?

Thanks.

**Comment by YannickNoelStephanKuhn** [comment]
Hi,
battery voltage basically functions as "Open-Circuit Voltage(charge state) - Overpotential(charge state, current)". Overpotential is always a loss of energy; hence it has the opposite sign to the current. I guess you set the Open-Circuit Voltage (OCV) to 0? I would recommend against that, as "simulating with OCV and removing it afterwards" leads to different results than "simulating without OCV".

I recommend the following articles for the basics of 1+1D battery models and their parameterization:
Why OCV is an essential factor: https://arxiv.org/abs/1702.02471
How to incorporate the electrolyte effects: https://arxiv.org/abs/1905.12553
